lib/gooddata/helpers/global_helpers_params.rb

```diff
@@ -264,9 +264,14 @@ def resolve_reference_params(data_params, params)
               elsif match =~ /\\\$/
                 '$'
               elsif match =~ /\$\{(\w+)\}/
-                val = params["#{$1}"] || raise("The gd_encoded_params parameter contains unknow reference #{$1}") # rubocop: disable Style/PerlBackrefs
-                reference_values << val
-                val
+                val = params["#{$1}"] # rubocop: disable Style/PerlBackrefs
```

Comment:
Please don't disable Rubocop rules in-line. Either disable the rule in the whole project or use `Regexp.last_match`.